generator/README.md

```diff
@@ -64,9 +64,17 @@ Dev mode is the way to use all new extensions from `master` branch:
 And `che:theia` will use `master` branch for all extensions and plugins, regardless of provided configuration
 
 ### Development life-cycle
-it's easy to check changes. Running yarn watch in a module and then running yarn watch in `examples/assembly` folder and using `yarn run start` in `examples/assembly`
+Che Theia should be built from root directory only (Root directory of Che Theia is the directory into which Theia was clonned and `che:theia init` was executed there). In case of building from subdirectories it will mess up dependencies, don't do it.
```

Comment:
```suggestion
Che Theia should be built from root directory only (Root directory of Che Theia is the directory into which Theia was cloned and `che:theia init` was executed there). In case of building from subdirectories it will mess up dependencies, don't do it.
```